revolv/project/models.py

```diff
@@ -302,17 +302,42 @@ def update_categories(self, category_list):
     def get_absolute_url(self):
         return reverse("project:view", kwargs={"pk": str(self.pk)})
 
+    def get_organic_donations(self):
+        return self.payment_set.filter(payment_type=PaymentType.objects.get_paypal())
+
+    def proportion_donated(self, user):
```

Comment:
Might be good to have a `:return:` here mentioning that it returns a float between 0 and 1 inclusive